tests/Common/AbstractMigrationTest.php

```diff
@@ -66,9 +37,13 @@ protected function tearDown(): void
      */
     public function testDropTable(): void
     {
-        Migration::dropTable($this->db);
+        Migration::ensureTable($this->db, '{{%custom-log}}');
```

Comment:
Now you test currently only custom table nam.

Need test both case. With default and with custom table names